packages/generator-adp/src/translations/adp-generator.i18n.json

```
@@ -0,0 +1,33 @@
+{
+    "yuiNavSteps": {
+        "flpCredentialsName": "FLP Configuration - Credentials",
+        "flpCredentialsDesc": "Enter credentials for your adaptation project's system ({{- system}})",
+        "sysConfirmDesc": "Validates connection to configured system",
+        "flpConfigName": "FLP Configuration",
+        "flpConfigDesc": "FLP Configuration for {{- projectName}}",
+        "configurationName": "Adaptation Project - Configuration"
+    },
+    "info": {
+        "flpConfigAdded": "FLP Configuration added successfully"
+    },
+    "prompts": {
+        "systemLabel": "Select System",
+        "systemTooltip": "Select a system to load its application list",
+        "usernameLabel": "Enter the user name for the backend system",
+        "passwordLabel": "Enter the password for the backend system",
+        "applicationListLabel": "Select Application",
+        "applicationListTooltip": "Select the application for which you want to create an app variant"
+    },
+    "error": {
+        "projectNotSupported": "Unable to find a supported adaptation project",
+        "fetchingManifest": "Error fetching merged manifest for base application",
+        "destinationNotFound": "Missing destination configuration in ui5.yaml",
+        "destinationNotInSubaccount": "Destination not found in the subaccount: {{- destination}}",
+        "systemNotFound": "Missing system configuration in ui5.yaml",
+        "systemNotFoundInStore": "System not found in the system store: {{- systemUrl}}",
+        "writingPhase": "Error in writing phase of the adaptation project FLP configuration",
```

```suggestion
        "writingPhase": "Error writing the adaptation project SAP Fiori launchpad configuration",
```

What should users try to fix this?

---

[2ca7bf8](https://github.com/SAP/open-ux-tools/pull/3031/commits/2ca7bf8848d2a9df8d1ee47f0aacf5fd4dd3fe1e). Although the text has changed (FLP part removed).

---

Okay. @nikmace. what can users do to fix the error? Should they try again? View the logs?